rapidsai/devel-centos.Dockerfile

```diff
@@ -82,6 +82,7 @@ RUN gpuci_mamba_retry create --no-default-packages --override-channels -n rapids
       -c gpuci \
       cudatoolkit=${CUDA_VER} \
       nvcc_linux-64=${CUDA_VER} \
+      openssl=1.1.1 \
```

Comment:
Wonder if we should also add a comment to the source code explaining this as well

Reply:
seems reasonable. I don't want to interrupt the current CI run, so I'll let that run through and add the comment after followed by a force merge.